Tests are failing

```text
Caused by: java.lang.ExceptionInInitializerError: Exception com.intellij.diagnostic.PluginException: 'ai.installer.notification.group.name' is not found (baseBundleName=messages.LLMInstallerBundle, bundle=com.intellij.IntelliJResourceBundle@24ff6b45) [Plugin: com.intellij.llmInstaller] [in thread "AWT-EventQueue-0"]
```

<details>
<summary>Full stacktrace</summary>

```text
com.intellij.testFramework.TestLoggerFactory$TestLoggerAssertionError: Could not initialize class com.jetbrains.python.sdk.pipenv.PipenvKt [Plugin: PythonCore]
	at com.intellij.testFramework.TestLoggerFactory$TestLogger.error(TestLoggerFactory.java:460)
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:436)
	at com.intellij.openapi.extensions.ExtensionPointName.forEachExtensionSafe(ExtensionPointName.kt:365)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor(EditorFactoryImpl.kt:222)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.kt:179)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.kt:144)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.doOpenTextEditor(TestEditorManagerImpl.kt:463)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileImpl3(TestEditorManagerImpl.kt:146)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.doOpenFile(TestEditorManagerImpl.kt:445)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openTextEditor(TestEditorManagerImpl.kt:436)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.createEditor(CodeInsightTestFixtureImpl.java:1720)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.lambda$configureInner$56(CodeInsightTestFixtureImpl.java:1661)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread$lambda$2(AnyThreadWriteThreadingSupport.kt:217)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread(AnyThreadWriteThreadingSupport.kt:216)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:842)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:395)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:446)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureInner(CodeInsightTestFixtureImpl.java:1636)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFileInner(CodeInsightTestFixtureImpl.java:1630)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFileInner(CodeInsightTestFixtureImpl.java:1612)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFilesInner(CodeInsightTestFixtureImpl.java:1511)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFile(CodeInsightTestFixtureImpl.java:1518)
	at insyncwithfoo.ryecharm.PlatformTestCase.fileBasedTest(PlatformTestCase.kt:34)
	at insyncwithfoo.ryecharm.others.dependencygroups.DependencyGroupInspectionTest.doTest(DependencyGroupInspectionTest.kt:33)
	at insyncwithfoo.ryecharm.others.dependencygroups.DependencyGroupInspectionTest.test circular group(DependencyGroupInspectionTest.kt:31)
	at java.base@21.0.5/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base@21.0.5/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.intellij.testFramework.TestLoggerKt$recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures$2.invoke(testLogger.kt:90)
	at com.intellij.testFramework.TestLoggerKt$recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures$2.invoke(testLogger.kt:90)
	at com.intellij.testFramework.TestLoggerKt.recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures(testLogger.kt:72)
	at com.intellij.testFramework.TestLoggerKt.recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures(testLogger.kt:90)
	at com.intellij.testFramework.UsefulTestCase.lambda$wrapTestRunnable$15(UsefulTestCase.java:533)
	at com.intellij.testFramework.UsefulTestCase.runTestRunnable(UsefulTestCase.java:426)
	at com.intellij.testFramework.fixtures.BasePlatformTestCase.runTestRunnable(BasePlatformTestCase.java:117)
	at com.intellij.testFramework.UsefulTestCase.defaultRunBare(UsefulTestCase.java:442)
	at com.intellij.testFramework.UsefulTestCase.lambda$runBare$13(UsefulTestCase.java:515)
	at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$5(EdtTestUtil.java:91)
	at com.intellij.openapi.application.TransactionGuardImpl.runWithWritingAllowed(TransactionGuardImpl.java:236)
	at com.intellij.openapi.application.TransactionGuardImpl.access$100(TransactionGuardImpl.java:25)
	at com.intellij.openapi.application.TransactionGuardImpl$1.run(TransactionGuardImpl.java:198)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread$lambda$2(AnyThreadWriteThreadingSupport.kt:217)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread(AnyThreadWriteThreadingSupport.kt:216)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:842)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:421)
	at com.intellij.openapi.application.impl.AppImplKt.rethrowExceptions$lambda$2(appImpl.kt:57)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:107)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:101)
	at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:27)
	at com.intellij.openapi.application.impl.AppImplKt.rethrowExceptions$lambda$3(appImpl.kt:68)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:102)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:117)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:43)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base@21.0.5/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base@21.0.5/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:675)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:573)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16$lambda$15(IdeEventQueue.kt:355)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:857)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18$lambda$17$lambda$16(IdeEventQueue.kt:354)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2$lambda$1(IdeEventQueue.kt:1045)
	at com.intellij.openapi.application.WriteIntentReadAction.lambda$run$0(WriteIntentReadAction.java:24)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:916)
	at com.intellij.openapi.application.WriteIntentReadAction.compute(WriteIntentReadAction.java:55)
	at com.intellij.openapi.application.WriteIntentReadAction.run(WriteIntentReadAction.java:23)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$2(IdeEventQueue.kt:1045)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:1054)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:117)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:1054)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$18(IdeEventQueue.kt:349)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:395)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
	Suppressed: com.intellij.openapi.util.TraceableDisposable$DisposalException: Editor EditorImpl[temp:///src/circularGroup/pyproject.toml (invalid, reason: File deleted: VfsEvent[deleted: temp:///src/circularGroup])] hasn't been released:
		at com.intellij.testFramework.RunAll.run(RunAll.kt:33)
		at com.intellij.testFramework.UsefulTestCase.tearDown(UsefulTestCase.java:328)
		at com.intellij.testFramework.fixtures.BasePlatformTestCase.tearDown(BasePlatformTestCase.java:73)
		at com.intellij.testFramework.UsefulTestCase.invokeTearDown(UsefulTestCase.java:455)
		at com.intellij.testFramework.UsefulTestCase.defaultRunBare(UsefulTestCase.java:439)
		... 50 more
	Caused by: com.intellij.openapi.util.TraceableDisposable$DisposalException: Editor EditorImpl[temp:///src/circularGroup/pyproject.toml (invalid, reason: File deleted: VfsEvent[deleted: temp:///src/circularGroup])] hasn't been released:
		at com.intellij.openapi.util.TraceableDisposable.throwDisposalError(TraceableDisposable.java:48)
		at com.intellij.openapi.editor.impl.EditorImpl.throwDisposalError(EditorImpl.java:1102)
		at com.intellij.openapi.editor.impl.EditorFactoryImpl$Companion.throwNotReleasedError(EditorFactoryImpl.kt:87)
		at com.intellij.testFramework.common.TestApplicationKt.checkEditorsReleased$lambda$21(testApplication.kt:289)
		at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
		at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:16)
		at com.intellij.testFramework.common.TestApplicationKt.checkEditorsReleased(testApplication.kt:297)
		at com.intellij.testFramework.LightPlatformTestCase.lambda$checkEditorsReleased$21(LightPlatformTestCase.java:485)
		at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
		at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
		at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
		at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:32)
		at com.intellij.testFramework.RunAll$Companion.runAll(RunAll.kt:45)
		at com.intellij.testFramework.RunAll.runAll(RunAll.kt)
		at com.intellij.testFramework.LightPlatformTestCase.checkEditorsReleased(LightPlatformTestCase.java:479)
		at com.intellij.testFramework.fixtures.impl.LightIdeaTestFixtureImpl.lambda$tearDown$7(LightIdeaTestFixtureImpl.java:89)
		at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
		at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
		at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
		at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:32)
		at com.intellij.testFramework.RunAll.run(RunAll.kt:21)
		at com.intellij.testFramework.fixtures.impl.LightIdeaTestFixtureImpl.tearDown(LightIdeaTestFixtureImpl.java:109)
		at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.lambda$tearDown$43(CodeInsightTestFixtureImpl.java:1469)
		at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$1(EdtTestUtil.java:51)
		at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
		at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:916)
		at com.intellij.testFramework.EdtTestUtil.runInEdtAndWait(EdtTestUtil.java:50)
		at com.intellij.testFramework.EdtTestUtil.runInEdtAndWait(EdtTestUtil.java:40)
		at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.lambda$tearDown$44(CodeInsightTestFixtureImpl.java:1469)
		at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
		at com.intellij.testFramework.RunAll$Companion$actionSequence$1$1.invoke(RunAll.kt:72)
		at com.intellij.testFramework.common.RunAllKt.runAllCatching(runAll.kt:56)
		at com.intellij.testFramework.common.RunAllKt.runAll(runAll.kt:32)
		at com.intellij.testFramework.RunAll$Companion.runAll(RunAll.kt:45)
		at com.intellij.testFramework.RunAll.runAll(RunAll.kt)
		at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.tearDown(CodeInsightTestFixtureImpl.java:1426)
		at com.intellij.testFramework.fixtures.BasePlatformTestCase.tearDown(BasePlatformTestCase.java:66)
		... 52 more
Caused by: com.intellij.diagnostic.PluginException: Could not initialize class com.jetbrains.python.sdk.pipenv.PipenvKt [Plugin: PythonCore]
	at com.intellij.serviceContainer.ComponentManagerImpl.createError(ComponentManagerImpl.kt:979)
	... 88 more
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.jetbrains.python.sdk.pipenv.PipenvKt
	at com.jetbrains.python.sdk.pipenv.PipEnvPipFileWatcher.isPipFileEditor(pipenv.kt:304)
	at com.jetbrains.python.sdk.pipenv.PipEnvPipFileWatcher.editorCreated(pipenv.kt:235)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor$lambda$1(EditorFactoryImpl.kt:222)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor$lambda$2(EditorFactoryImpl.kt:222)
	at com.intellij.openapi.extensions.ExtensionPointName.forEachExtensionSafe(ExtensionPointName.kt:61)
	... 87 more
Caused by: java.lang.ExceptionInInitializerError: Exception com.intellij.diagnostic.PluginException: 'ai.installer.notification.group.name' is not found (baseBundleName=messages.LLMInstallerBundle, bundle=com.intellij.IntelliJResourceBundle@24ff6b45) [Plugin: com.intellij.llmInstaller] [in thread "AWT-EventQueue-0"]
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance$suspendImpl(ServiceInstanceInitializer.kt:52)
	at com.intellij.serviceContainer.ServiceInstanceInitializer.createInstance(ServiceInstanceInitializer.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invokeSuspend(LazyInstanceHolder.kt:163)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:62)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:163)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invokeSuspend(LazyInstanceHolder.kt:161)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder$initialize$1.invoke(LazyInstanceHolder.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startCoroutineUndispatched(Undispatched.kt:27)
	at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:90)
	at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:123)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:52)
	at kotlinx.coroutines.BuildersKt.launch(Unknown Source)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.initialize(LazyInstanceHolder.kt:146)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.access$initialize(LazyInstanceHolder.kt:14)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.tryInitialize(LazyInstanceHolder.kt:136)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstance(LazyInstanceHolder.kt:96)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext$suspendImpl(LazyInstanceHolder.kt:88)
	at com.intellij.platform.instanceContainer.internal.LazyInstanceHolder.getInstanceInCallerContext(LazyInstanceHolder.kt)
	at com.intellij.serviceContainer.ComponentManagerImplKt$doGetOrCreateInstanceBlocking$1.invokeSuspend(ComponentManagerImpl.kt:1566)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:277)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:111)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$BuildersKt__BuildersKt(Builders.kt:84)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:52)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization$lambda$10(ComponentManagerImpl.kt:1684)
	at com.intellij.openapi.progress.ContextKt.prepareThreadContext(context.kt:82)
	at com.intellij.serviceContainer.ComponentManagerImplKt.runBlockingInitialization(ComponentManagerImpl.kt:1675)
	at com.intellij.serviceContainer.ComponentManagerImplKt.doGetOrCreateInstanceBlocking(ComponentManagerImpl.kt:1565)
	at com.intellij.serviceContainer.ComponentManagerImplKt.getOrCreateInstanceBlocking(ComponentManagerImpl.kt:1560)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:751)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:695)
	at com.intellij.notification.NotificationGroupManager.getInstance(NotificationGroupManager.java:19)
	at com.jetbrains.python.sdk.pipenv.PipenvKt.LOCK_NOTIFICATION_GROUP$lambda$4(pipenv.kt:320)
	at com.intellij.openapi.progress.Cancellation.computeInNonCancelableSection(Cancellation.java:114)
	at com.intellij.openapi.progress.Cancellation.forceNonCancellableSectionInClassInitializer(Cancellation.java:165)
	at com.jetbrains.python.sdk.pipenv.PipenvKt.<clinit>(pipenv.kt:319)
	at com.jetbrains.python.sdk.pipenv.PipEnvPipFileWatcher.isPipFileEditor(pipenv.kt:304)
	at com.jetbrains.python.sdk.pipenv.PipEnvPipFileWatcher.editorCreated(pipenv.kt:235)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor$lambda$1(EditorFactoryImpl.kt:222)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor$lambda$2(EditorFactoryImpl.kt:222)
	at com.intellij.openapi.extensions.ExtensionPointName.forEachExtensionSafe(ExtensionPointName.kt:61)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.doCreateEditor(EditorFactoryImpl.kt:222)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.kt:179)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.kt:144)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.doOpenTextEditor(TestEditorManagerImpl.kt:463)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openFileImpl3(TestEditorManagerImpl.kt:146)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.doOpenFile(TestEditorManagerImpl.kt:445)
	at com.intellij.openapi.fileEditor.impl.TestEditorManagerImpl.openTextEditor(TestEditorManagerImpl.kt:436)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.createEditor(CodeInsightTestFixtureImpl.java:1720)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.lambda$configureInner$56(CodeInsightTestFixtureImpl.java:1661)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread$lambda$2(AnyThreadWriteThreadingSupport.kt:217)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runIntendedWriteActionOnCurrentThread(AnyThreadWriteThreadingSupport.kt:216)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:842)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:395)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:446)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureInner(CodeInsightTestFixtureImpl.java:1636)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFileInner(CodeInsightTestFixtureImpl.java:1630)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFileInner(CodeInsightTestFixtureImpl.java:1612)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFilesInner(CodeInsightTestFixtureImpl.java:1511)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.configureByFile(CodeInsightTestFixtureImpl.java:1518)
	at insyncwithfoo.ryecharm.PlatformTestCase.fileBasedTest(PlatformTestCase.kt:34)
	at insyncwithfoo.ryecharm.others.dependencygroups.DependencyGroupInspectionTest.doTest(DependencyGroupInspectionTest.kt:33)
	at insyncwithfoo.ryecharm.others.dependencygroups.DependencyGroupInspectionTest.test non-normalized circular group(DependencyGroupInspectionTest.kt:13)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.intellij.testFramework.TestLoggerKt$recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures$2.invoke(testLogger.kt:90)
	at com.intellij.testFramework.TestLoggerKt$recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures$2.invoke(testLogger.kt:90)
	at com.intellij.testFramework.TestLoggerKt.recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures(testLogger.kt:72)
	at com.intellij.testFramework.TestLoggerKt.recordErrorsLoggedInTheCurrentThreadAndReportThemAsFailures(testLogger.kt:90)
	at com.intellij.testFramework.UsefulTestCase.lambda$wrapTestRunnable$15(UsefulTestCase.java:533)
	at com.intellij.testFramework.UsefulTestCase.runTestRunnable(UsefulTestCase.java:426)
	at com.intellij.testFramework.fixtures.BasePlatformTestCase.runTestRunnable(BasePlatformTestCase.java:117)
	at com.intellij.testFramework.UsefulTestCase.defaultRunBare(UsefulTestCase.java:442)
	at com.intellij.testFramework.UsefulTestCase.lambda$runBare$13(UsefulTestCase.java:515)
	at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$3(EdtTestUtil.java:80)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:128)
	at com.intellij.testFramework.EdtTestUtil.lambda$runInEdtAndWait$4(EdtTestUtil.java:79)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
	at com.intellij.util.concurrency.ChildContext$runInChildContext$1.invoke(propagation.kt:101)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:107)
	at com.intellij.util.concurrency.ChildContext.runInChildContext(propagation.kt:101)
	at com.intellij.util.concurrency.ContextRunnable.run(ContextRunnable.java:27)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:308)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:781)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:750)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:318)
	... 6 more
```
</details>

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -61,7 +61,12 @@ dependencies {
         )
         
         // Plugin Dependencies. Uses `platformBundledPlugins` property from the gradle.properties file for bundled IntelliJ Platform plugins.
-        bundledPlugins(providers.gradleProperty("platformBundledPlugins").map { it.split(',') })
+        bundledPlugins(
+            providers.gradleProperty("platformBundledPlugins").map {
+                // TODO: Remove `com.intellij.llmInstaller` once 2024.3.1 is no longer supported
+                it.split(',').plus("com.intellij.llmInstaller")
+            }
+        )
         
         // Plugin Dependencies. Uses `platformPlugins` property from the gradle.properties file for plugin from JetBrains Marketplace.
         plugins(providers.gradleProperty("platformPlugins").map { it.split(',') })

